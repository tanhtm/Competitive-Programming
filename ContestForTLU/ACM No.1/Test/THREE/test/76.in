796800661000000000
