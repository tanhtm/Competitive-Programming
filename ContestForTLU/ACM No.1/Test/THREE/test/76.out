1 2 5 7 13 15 16 19 25 33 36 37 
0 3 4 9 10 14 17 18 20 24 26 32 35 38 
